packages/ui-react/src/styles/_theme.scss

```diff
@@ -1,5 +1,7 @@
 @use 'variables';
 
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400..700&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
```

Comment:
I think Dancing Script is not necessary here

Reply:
Whoops! I included the wrong font. This is fixed.